modules/persistent-storage-csi-efs-metrics-procedure-gui.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+[id="efs-metrics-procedure-gui_{context}"]
+= Enabling usage metrics using the web console
+
+To enable AWS EFS storage CSI usage metrics using the web console:
+
+. Click *Administration* > *CustomResourceDefinitions*.
+
+. On the *CustomResourceDefinitions* page next to the *Name* dropdown box, type "clustercsidriver".
```

Comment:
This looks weird with quotes. I see a few instances like the following in the docs:
```suggestion
. On the *CustomResourceDefinitions* page next to the *Name* dropdown box, type `clustercsidriver`.
```